readme.md

```diff
@@ -40,13 +40,20 @@ Pact.crypto.toTweetNaclSecretKey(keyPair) -> <Uint8Array>
 ### Language Expression Construction
 
 A helper function for constructing native Pact commands.
-
-```
-Pact.lang.mkExp(<string>, *args) -> <string>
+- mkExp takes in Pact function and its arguments and returns a Pact expression.
+- mkMeta returns gas information of the tx in object format. This is only important for the txs in public blockchain. txs don't need gas in private blockchain.
+   * "sender" represents the gas account, and the tx must be signed with the keyset associated with the gas account. Otherwise, the tx will be rejected.
+   * "ChainId" represents the Chain Id that the tx will be sent to.
+   * "gasPrice" represents the gas price of the tx.
+   * "gasLimit" represents the gas limit of the tx.
+   * "creationTime" represents the creation time of the tx (0 refers to current time).
```

Comment:
hmm, is this true about `0`? see my comment on other PR

Reply:
@emilypi Could you also confirm this? Thanks!

Reply:
Confirmed and edited to 
```
   * "creationTime" represents the tx's wait time to be considered a candidate for inclusion into a block in the blockchain. (in seconds)
   * "ttl" represents the tx's maximum wait time to be considered a candidate for inclusion into a block in the blockchain. (in seconds)
```